SnapChat/App/Components/Me/Me.js

```diff
@@ -1,9 +1,90 @@
-import React from 'react'
-import { View } from 'react-native'
+import React, { PropTypes } from 'react'
+import { View, Text, Image, TouchableHighlight } from 'react-native'
 import { meStyles as styles } from './meStyles'
 
-const Me = () => (
-  <View />
+const Me = ({
+    addedMePressed,
+    addFriendsPressed,
+    myFriendsPressed,
+    cameraBackPressed,
+    trophyCasePressed,
+    settingsPressed
+    }) => (
+
+    <View style={styles.container}>
+
+        {/* Header */}
+        <View style={styles.header}>
+            <TouchableHighlight>
+                <Text style={styles.headerButton}> / ? / </Text>
+            </TouchableHighlight>
+            <TouchableHighlight style={styles.flex} onPress={trophyCasePressed}>
+                <Text style={styles.headerMiddle}> / T / </Text>
+            </TouchableHighlight>
+            <TouchableHighlight onPress={settingsPressed}>
+                <Text style={styles.headerButton}> / S / </Text>
+            </TouchableHighlight>
+        </View>
+
+        {/* Barcode Image */}
+        <View style={styles.image}>
+            <Image
+                style={styles.barcodeImage}
+                source={require('../../../images/barcodeImage.png')}
+            />
+            <Text style={styles.name}>Ryan O'Kane</Text>
```

Comment:
i was just mentioning, if you look at the full file, after the apostrophe it changed the whole github file to be highlighted red. I don't think you can fix it / not a real issue